samples/net/zperf/src/zperf_shell.c

```diff
@@ -1066,7 +1066,7 @@ static void zperf_init(const struct shell *shell)
 		if (ret < 0) {
 			shell_fprintf(shell, SHELL_WARNING,
 				      "Unable to set IP %s\n",
-				      DST_IP6ADDR);
+				      DST_IP6ADDR ? DST_IP6ADDR : "");
```

Comment:
This probably should not even run when `DST_IP6ADDR == NULL`?

Reply:
Just checked back on this: I could hide this code behind an #if, but this would go against the spirit of the code a used a few lines above:

```C
	if (IS_ENABLED(CONFIG_NET_IPV4) && MY_IP4ADDR) {
```

Do you think I should rework this line?

Reply:
I think we could just add an additional check - if `DST_IP6ADDR` is not set, print a warning, something like `Default IPv6 destination address not set`, otherwise proceed with parsing. The same would apply to IPv4 just below.

Reply:
I pushed a change which implements this. However, in my understanding, it will actually never ever be printed.

Reply:
Unless someone changes the sample configuration.